source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2668,7 +2668,7 @@ export class Media extends CardElement {
         return element;
     }
 
-    static onPlay: (sender: Media) => void;
+    static onPlay: (sender: Media) => void = null;
```

Comment:
Can you elaborate why this is necessary here only? There are other event declarations in the library that also do not include `= null`. Also why `null` and not `undefined`?

Reply:
I got the error `Only initialized variables and constants can be referenced because the value of this variable is needed by the template compiler`
This is the only static field that is not initialized in this class.
You're right: `undefined ` would be better.

Reply:
i was wrong about the `undefined`. I took again a look in at the code and the reason I used `null` is because of the click() method on this class. There is a if-statement that checks for `this.onClick != null` . So undefined would fail the test and it would the invocation of the method `this.onClick(this);` in the next line would fail.